### PR TITLE
Add event struct and type for Addon Update Status

### LINKED ISSUE #149: [Events] Add UpdateAddonStatus event

### PR DESCRIPTION
**Pull request by brandon-welsch**
Close #149

### CODE REVIEW

File: events_structs.go [hunk 1]
```diff
@@ -816,6 +818,22 @@ func (ev *EventDeleteAutoscalerType) String() string {
 	return fmt.Sprintf("Alert deleted about %s on container %s", d.Metric, d.ContainerType)
 }
 
+type EventAddonUpdatedTypeData struct {
+	EventAddon
+	Status            []AddonStatus     `json:"status"`
+	AttributesChanged map[string]string `json:"attributes_changed"`
```
Comment (Soulou) [inline events_structs.go]: Why a hash, what's inside, can you set an example of the payload?

Comment (brandon-welsch) [reply events_structs.go]: It's a mistake, an example of the payload is: `['status', 'app_id', ...]` and not `[status: [...], app_id: [...]]`

**Comment by Soulou** [comment]
My comment about an example still stands you know ;-)

**Comment by brandon-welsch** [comment]
Payload example for the event :
```ruby
{
    "id":"5dcd7234597cb1009833010b",
    "created_at":"2019-11-14T15:26:44.600Z",
    "type":"addon_updated",
    "app_id":"5dcd1e76597cb100014eed4c",
    "app_name":"test-addon-yes-pr2",
    "type_data":{
        "status":["provisioning", "running"],
        "attributes_changed":["status"],
        "addon_provider_name":"Scalingo Redis",
        "plan_name":"sandbox",
        "resource_id":"test-addon-yes-pr2-2062"
    }
}
```

**Comment by Soulou** [comment]
LGTM

**Pull request merged**